ci_framework/roles/openshift_provisioner_node/tasks/add_virtual_network.yml

```diff
@@ -0,0 +1,28 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Rendering the virtual network template
+  vars:
+    virt_net_name: "{{ item }}"
+  ansible.builtin.template:
+    src: templates/virt_net.xml.j2
+    dest: "/tmp/{{ item }}.xml"
+    mode: 0644
+
+- name: Define and autostart network
+  become: true
+  ansible.builtin.script:
+    cmd: files/add_virt_net.sh "{{ item }}"
```

Comment:
maybe we can rely on the actual `community.libvirt` collection?

Reply:
Initially, the implementation was done with `community.libvirt.virt_net` collection however the rendering of the template threw an `lxml` error. Hence used the easy way out.

```
- name: Define and autostart network
  become: true
  community.libvirt.virt_net:
    command: define
    autostart: true
    state: present
    xml: '{{ lookup("template", "templates/virt_net.xml.j2") }}'
```

Reply:
Will try to debug and modify accordingly.

Reply:
Thanks for the push. The host was missing `python3-lxml`. Fixed in the latest patch set.